src/platform/plugins/shared/dashboard/server/api/register_routes.ts

```diff
@@ -222,17 +222,15 @@ export function registerAPIRoutes({
               min: 1,
               defaultValue: 1,
             }),
-            perPage: schema.maybe(
-              schema.number({
-                meta: {
-                  description:
-                    'The number of dashboards to display on each page (max 1000). Default is "20".',
-                },
-                defaultValue: 20,
-                min: 1,
-                max: 1000,
-              })
-            ),
+            perPage: schema.number({
+              meta: {
+                description:
+                  'The number of dashboards to display on each page (max 1000). Default is "20".',
+              },
+              defaultValue: 20,
+              min: 1,
+              max: 1000,
+            }),
```

Comment:
Using a `defaultValue` is enough to treat this as optional, but when also paired with `schema.maybe` the validated value will be `undefined` instead of `20`.

Reply:
Mostly for @nickpeihl